tools/gomod-required-updater/internal/updater/system_operator.go

```diff
@@ -0,0 +1,65 @@
+package updater
+
+import (
+	"fmt"
+	"io"
+	"io/fs"
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+type systemOperator struct {
+	stdout io.Writer
+	stderr io.Writer
+}
+
+func NewSystemOperator() SystemOperator {
+	return &systemOperator{
+		stdout: os.Stdout,
+		stderr: os.Stderr,
+	}
+}
+
+// For testing
+func NewSystemOperatorWithIO(stdout, stderr io.Writer) SystemOperator {
```

Comment:
Might be best to name this with `Mock` if it's for testing, and declare it in the test file

Reply:
Was able to safely remove it b5dc9fb4344836c3e4d8d15619acbe378f465dba.